### PR TITLE
less: Added tailf example

### PR DESCRIPTION
**Pull request by aniket-deole**
F

----

~~- [ ] The page (if new), does not already exist in the repo.~~

~~- [ ] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.~~

- [*] The page has 8 or fewer examples.

- [*] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [*] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/less.md [hunk 1]
```diff
@@ -22,6 +22,10 @@
 
 `?{{something}}`
 
+- Tail currently opened file:
```
Comment (agnivade) [inline pages/common/less.md]: How about - `Follow the output of the currently opened file`

File: pages/common/less.md [hunk 2]
```diff
@@ -22,6 +22,10 @@
...
+
+`<Shift> F`
```
Comment (agnivade) [inline pages/common/less.md]: Just `F` is sufficient. Above in the page it's mentioned `G` and `g`.

**Comment by agnivade** [comment]
Please sign the CLA too.

File: pages/common/less.md (hunk 1 above)
Comment (aniket-deole) [reply pages/common/less.md]: Done

File: pages/common/less.md (hunk 2 above)
Comment (aniket-deole) [reply pages/common/less.md]: Done

**Comment by sbrl** [comment]
Thanks, @aniket-deole!

**Pull request merged**